docs/reference/vpc-association-policy.md

```diff
@@ -0,0 +1,85 @@
+# VpcAssociationPolicy API Reference
+
+## VpcAssociationPolicy
+
+VpcAssociationPolicy is a CRD that can be attached to a Gateway to define the ServiceNetworkVpcAssociation configuration.
+
+
+
+One of its option is `securityGroupIds`. it can control the inbound traffic from current cluster workloads to the gateway listeners. Please check the VPC lattice doc for more detail of this option. https://docs.aws.amazon.com/vpc-lattice/latest/ug/security-groups.html
```

Comment:
By options, does this mean properties of the policy CRD? Can we create a section of all the properties in a CRD and break down what each property does?

Reply:
Also, when linking to a doc, let's use hyperlinks instead of pasting a raw link

Reply:
Will change